translations/fr/pack/parallel/ltr_encounter.json

```
@@ -16,7 +16,7 @@
   },
   {
     "back_name": "La Tempête Spectrale",
-    "back_flavor": "De doux chants vous guident vers une clairière obscure près de la chapelle abandonnée. Un cercle d’herbes sorcières entoure la clairière, se balançant doucement au gré de la brise. Au centre de la clairière, une femme masquée s'agenouille devant une tombe sans nom, éloignée du reste du cimetière. Vous observez quelques minutes la femme chanter une mélodie ancienne et sombre, rappelant un peu une berceuse d'enfant — apaisante, mais avec une gravité sombre qui trahit son objectif. Dès la fin de la chanson, le manteau de la femme se dissout dans la brume et elle disparaît dans un tourbillon d'ombres.",
+    "back_flavor": ,
```

This is invalid JSON. If you mean to empty the text, please replace with this suggestion or remove the line altogether.

```suggestion
    "back_flavor": "",
```